cmd/soroban-rpc/internal/test/cli_test.go

```diff
@@ -0,0 +1,67 @@
+package test
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/creachadair/jrpc2"
+	"github.com/creachadair/jrpc2/jhttp"
+	"github.com/stellar/go/keypair"
+	"github.com/stellar/go/txnbuild"
+	"github.com/stellar/go/xdr"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInstallContractWithCLI(t *testing.T) {
+	NewCLITest(t)
+	output, err := runCLICommand(t, "contract install --wasm ../../../../target/wasm32-unknown-unknown/test-wasms/test_hello_world.wasm")
+	require.NoError(t, err)
+	wasm := getHelloWorldContract(t)
+	contractHash := xdr.Hash(sha256.Sum256(wasm))
+	require.Contains(t, string(output), contractHash.HexString())
+}
+
+func runCLICommand(t *testing.T, cmd string) ([]byte, error) {
+	baseCmdArgs := []string{"run", "--", "--vv"}
+	args := strings.Split(cmd, " ")
+	args = append(baseCmdArgs, args...)
+	c := exec.Command("cargo", args...)
+	c.Env = append(os.Environ(),
+		fmt.Sprintf("RPC_URL=http://localhost:%d/", sorobanRPCPort),
+		fmt.Sprintf("NETWORK_PASPRHASE=%s", StandaloneNetworkPassphrase),
```

Comment:
would that work if the environment already includes these ? I think that you'd want to override existing entries, if they exist.